src/event/event_epoll.c

```diff
@@ -194,7 +202,7 @@ _dispatch_unote_register(dispatch_unote_t du, dispatch_priority_t pri)
 	case DISPATCH_EVFILT_CUSTOM_ADD:
 	case DISPATCH_EVFILT_CUSTOM_OR:
 	case DISPATCH_EVFILT_CUSTOM_REPLACE:
-		du._du->du_wlh = DISPATCH_WLH_GLOBAL;
+		du._du->du_wlh = wlh;
```

Comment:
ok;  I think I should then add an assert that the value of the wlh parameter is DISPATCH_WLH_GLOBAL.  I don't like ignoring the value the caller passed in even without asserting it has a known value.